app/src/app_config.py

```diff
@@ -19,3 +25,13 @@ class AppConfig(PydanticBaseEnvConfig):
     host: str = "127.0.0.1"
     port: int = 8080
     chat_engine: str = "guru-snap"
+
+    def db_session(self) -> db.Session:
+        return db.PostgresDBClient().get_session()
+
+    @cached_property
+    def sentence_transformer(self) -> SentenceTransformer:
+        return SentenceTransformer(self.embedding_model)
+
+
+app_config = AppConfig()
```

Comment:
Assuming `AppConfig` only ever needs to be instantiated once per application startup.